nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -87,41 +81,49 @@ def transform(self, transformation_layout: TransformationLayout) -> ov.Model:
                 model_extraction_transformation = transformation
             elif isinstance(transformation, OVBiasCorrectionCommand):
                 bias_correction_transformations.append(transformation)
-
-        if output_insertion_transformations:
-            self._apply_output_insertion_transformations(output_insertion_transformations)
+        # Inplace transformations; Using deepcopy of model
         if fq_nodes_removing_transformations:
-            self._apply_fq_nodes_removing_transformation(fq_nodes_removing_transformations)
-        if quantizer_insertion_transformations:
-            self._apply_quantizer_insertion_transformations(quantizer_insertion_transformations)
-        if bias_correction_transformations:
-            self._apply_bias_correction_transformations(bias_correction_transformations)
-        if model_extraction_transformation:
-            self._model = self._apply_model_extraction_transformation(model_extraction_transformation)
-
-        return self._model
+            model = self._apply_fq_nodes_removing_transformation(fq_nodes_removing_transformations)
+        elif quantizer_insertion_transformations:
+            model = self._apply_quantizer_insertion_transformations(quantizer_insertion_transformations)
+        elif bias_correction_transformations:
+            model = self._apply_bias_correction_transformations(bias_correction_transformations)
+        elif model_extraction_transformation:
+            model = self._apply_model_extraction_transformation(model_extraction_transformation)
+        elif output_insertion_transformations:
+            model = self._apply_output_insertion_transformations(output_insertion_transformations)
+        # No transformation applied
+        else:
+            nncf_logger.warning('No transformations were applied to the model. The copy of the model is returned.')
```

Comment:
It is not a warning. It is the default behavior. Could you please describe such behavior in the docstring?

Reply:
It should be removed.

Reply:
Done